contracts/tools/Swaps.sol

```diff
@@ -381,103 +437,128 @@ contract DVP is Ownable, ERC1820Client, IERC1400TokensRecipient, ERC1820Implemen
    * @param ethValue Value sent (only used for ETH)
    * @param erc1400TokenValue Value sent (only used for ERC1400)
    */
-  function _acceptTrade(uint256 index, address sender, uint256 ethValue, uint256 erc1400TokenValue) internal {
+  function _acceptTrade(uint256 index, address sender, uint256 ethValue, uint256 erc1400TokenValue, bytes32 preimage) internal {
     Trade storage trade = _trades[index];
     require(trade.state == State.Pending, "Trade is not pending");
 
+    address recipientHolder;
     address selectedHolder;
```

Comment:
Just noticed that "selectedHolder" is always equal to "sender".
- Either we can remove that variable.
- Or if we want to keep it for readability reasons, we shall use "selectedHolder" instead of "sender" below